src/_includes/partials/token_table.njk

```diff
@@ -0,0 +1,58 @@
+<table class="table-display-token-{{token}}">
+<tr>
+{% for header in headers %}
+<th>{{ header[1] }}</th>
+{% endfor %}
+</tr>
+{% comment %}
+tokens: all the tokens
+token: key
+filteredTokens: the filtered tokens
+subTokens: part of the filtered tokens
```

Comment:
I think these are understandable, but I'm open to suggestion to make these clearer